src/openeo_grass_gis_driver/app.py

```diff
@@ -11,7 +11,6 @@
 
 flask_app = Flask(__name__)
 flask_app.wsgi_app = ProxyFix(flask_app.wsgi_app, x_for=1, x_host=1, x_proto=1)
-# as flask CORS lead to unforeseen behaviour in certain cases, we configure
```

Comment:
Have you tested these "certain cases"?

Reply:
The certain cases do not apply any more, see second part of https://github.com/Open-EO/openeo-grassgis-driver/pull/85#issuecomment-829236931.